crates/bevy_ui/src/ui_node.rs

```diff
@@ -1844,22 +1844,31 @@ impl TargetCamera {
     }
 }
 
+#[derive(Component)]
+/// Marker used to demarcate default cameras, they will have priority over `PrimaryWindow` cameras
```

Comment:
```suggestion
/// Marker used to identify default cameras, they will have priority over `PrimaryWindow` cameras.
```

As a non-native English speaker, I had to google "demarcate" to understand this.